python/paddle/fluid/dygraph/checkpoint.py

```diff
@@ -16,13 +16,15 @@
 
 import os
 import collections
-from .. import core
 from ..framework import Variable, default_main_program
+import pickle
+from . import learning_rate_scheduler
+import warnings
 
 __all__ = ['save_persistables', 'load_persistables']
 
 
-def save_persistables(vardict, dirname, filename=None):
+def save_persistables(model_dict, optimizer, dirname, filename=None):
```

Comment:
if optimizer is none？

Reply:
it can be solved at line 132

> if optimizer is none？

I think it can be solved at line 132